src/state/slices/tradeQuoteSlice/selectors.ts

```diff
@@ -394,7 +393,7 @@ export const selectPotentialDonationAmountUserCurrency: Selector<ReduxState, str
       if (activeQuote?.affiliateBps === undefined) return undefined
       else {
         const affiliatePercentage = convertBasisPointsToDecimalPercentage(
-          DEFAULT_SWAPPER_DONATION_BPS,
+          activeQuote.affiliateBps ?? '0',
```

Comment:
this was resulting in showing 30 bps donation when it was 0 for a non-donating quote, or 25 for a thorchain quote 💀